ci/travis/build-docker-images.py

```diff
@@ -230,6 +230,10 @@ def push_and_tag_images(push_base_images: bool, merge_build: bool = False):
         DOCKER_CLIENT.api.login(username=username, password=password)
 
     def docker_push(image, tag):
+        # Do not tag release builds because they are no longer up to
+        # date after the branch cut.
+        if "nightly" in tag and _release_build():
```

Comment:
why do you check nightly in tag?

Reply:
Because the release is technically **behind** the latest master, so we don't want to push to `nightly` when on a release branch.